Deployment/terraform_conditional_run.ps1

```diff
@@ -0,0 +1,58 @@
+param (
+    [Parameter(Mandatory = $true)] [string] $DeploymentResourceGroupName,
+    [Parameter(Mandatory = $true)] [string] $DeploymentStorageAccountName,
+    [Parameter(Mandatory = $true)] [string] $WorkSpace,
+    [Parameter(Mandatory = $true)] [string] $Terraformapply
```

Comment:
Please can we have a more readable/meaningful variable name.
```suggestion
    [Parameter(Mandatory = $true)] [string] $ContinueWhenResourceToBeDestroyed
```

Reply:
Changes variable name from Terraformapply to ContinueEvenIfResourcesAreGettingDestroyed